src/lib/backend_fapi.c

```diff
@@ -16,6 +17,12 @@
 FAPI_CONTEXT *fctx = NULL;
 unsigned maxobjectid = 0;
 
+static void fail_fapi_msg(TSS2_RC rc) {
+    LOGW("Listing FAPI token objects failed: \"%s\"\n"
+            "Please see https://github.com/tpm2-software/tpm2-pkcs11/blob/master/docs/FAPI.md "
```

Comment:
Instead of using `.../master/...`, would it be possible to use `VERSION` macro? For example:

```c
    LOGW("Listing FAPI token objects failed: \"%s\"\n"
            "Please see https://github.com/tpm2-software/tpm2-pkcs11/blob/%s/docs/FAPI.md "
            "for more details", VERSION, Tss2_RC_Decode(rc));
```

or

```c
#ifdef VERSION
    LOGW("Listing FAPI token objects failed: \"%s\"\n"
            "Please see https://github.com/tpm2-software/tpm2-pkcs11/blob/master/docs/FAPI.md "
            "for more details", VERSION, Tss2_RC_Decode(rc));
#else
    LOGW("Listing FAPI token objects failed: \"%s\"\n"
            "Please see https://github.com/tpm2-software/tpm2-pkcs11/blob/%s/docs/FAPI.md "
            "for more details", VERSION, Tss2_RC_Decode(rc));
 #endif
```